recipes/onnxruntime/all/patches/1.15.0-0001-cmake-dependencies.patch

```diff
@@ -0,0 +1,213 @@
+diff --git a/cmake/external/abseil-cpp.cmake b/cmake/external/abseil-cpp.cmake
+index 54d2f9c5c1..3195ef15a0 100644
+--- a/cmake/external/abseil-cpp.cmake
++++ b/cmake/external/abseil-cpp.cmake
+@@ -22,6 +22,7 @@ FetchContent_Declare(
+     URL ${DEP_URL_abseil_cpp}
+     URL_HASH SHA1=${DEP_SHA1_abseil_cpp}
+     PATCH_COMMAND ${ABSL_PATCH_COMMAND}
++    FIND_PACKAGE_ARGS REQUIRED CONFIG NAMES absl
```

Comment:
It is really necessary, the package won't build otherwise.
It is the "same" patch as `1.14.1-0002-cmake-dependencies.patch` with one exception:
```
-  FIND_PACKAGE_ARGS 3.21.12 NAMES Protobuf
+  FIND_PACKAGE_ARGS NAMES Protobuf
```

`3.21.12` shouldn't be required upstream but it's there so until #17725 gets merged, this is required as well.

The previous patch did not apply cleanly so I had to re-create it (hence the exception being included in the 1.15.0 patch, would have kept the 1.14.1 patch otherwise and patched the exception over that).

I created a PR upstream for the ones that were in `1.14.1-0001-cmake-dependencies.patch` to reduce the amount of patching required in conan and that patch is gone in 1.15.0